app/views/podcasts/index.html.erb

```diff
@@ -6,15 +6,23 @@
       <div class="d-grid d-lg-flex gap-2 align-items-center">
         <h1 class="text-black fw-bold d-flex flex-fill my-4"><%= t(".my_podcasts") %></h1>
 
-        <a data-bs-toggle="dropdown">
-        <button class="btn btn-outline-light btn-sm dropdown-toggle">
-          Sort <span class="material-icons" style="color: green" aria-hidden="true" aria-hidden="true">sort</span>
+        <button class="btn btn-outline-light btn-sm dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+        <span class="material-icons" style="color: green" aria-hidden="true" aria-hidden="true">sort</span>
+          <% if params[:sort] == 'episode_count' %>
+            <span class="material-icons" aria-hidden="true">123</span>
+          <% elsif params[:sort] == 'A-Z' %>
+            <span>A-Z</span>
+          <% elsif params[:sort] == 'Z-A' %>
+            <span>Z-A</span>
+          <% else %>
+            <span class="material-icons" aria-hidden="true">timer</span>
+          <% end %>
         </button>
         <ul class="dropdown-menu">
-          <li><%= link_to "Recent Activity", podcasts_path(sort: "Recent"), class: "dropdown-item active" %></li>
-          <li><%= link_to "# of Episodes", podcasts_path(sort: "episode_count"), class: "dropdown-item" %></li>
-          <li><%= link_to "A-Z", podcasts_path(sort: "A-Z"), class: "dropdown-item" %></li>
-          <li><%= link_to "Z-A", podcasts_path(sort: "Z-A"), class: "dropdown-item" %></li>
+        <li><%= active_link_to "Recent Activity", podcasts_path(sort: "Recent Activity"), active: :exact, class: "dropdown-item text-decoration-none" %></li>
```

Comment:
I think these should stay indented, inside the `<ul>`?

Reply:
Huh. It's indented in my VS code. Would linter change that??